README.md

```diff
@@ -76,32 +100,83 @@ For instance, `wnfs/APPEND` is an action for WebNative filesystem paths. The act
+
+``` plaintext
+                 ┌───────────────────┐  ─┐
+                 │                   │   │
+                 │                   │   │
+┌────────────────┼───┐               │   │
+│                │   │ Resource B    │   │     
+│                │   │               │   │       BxZ
+│                │   │     X         │   ├─── Capability
+│     Resource A │   │               │   │
+│                │   │ Ability Z     │   │
+│         X      │   │               │   │
+│                │   │               │   │
+│     Ability Y  │   │               │   │
+│                └───┼───────────────┘  ─┘
+│                    │
+│                    │
+└────────────────────┘
+
+└──────────────────┬─────────────────┘
+                   │
+
+               AxY U BxZ
+                 scope
+```
+
+The capability scope is the total rights of the authorization space down to the relevant volume of authorizations. Individual capabilities MAY overlap; the scope is the union. With the exception of [rights amplification](#54-rights-amplification), every individual delegation MUST have equal or narrower capabilties from their delegator. Inside this content space, you can draw a boundary around some resource(s) (their type, identifiers, and paths or children), and their capabilities.
+
+For example, given the following scopes against a WebNative filesystem, they can be merged as follows:
 
-An authorization scope is a set of capabilities. Scopes MUST compose with set semantics, so multiple scopes in an array MAY be considered the (deduplicated) union of all of the inner scopes.
+```js
+// "wnfs" abilities:
+// FETCH < APPEND < OVERWRITE < SUPERUSER
 
-![](./assets/scope_union.jpg)
+ScopeA = [
+  { "with": "wnfs://alice.example.com/pictures/", "can": "wnfs/APPEND" }
+];
 
-The "scope" is the total rights of the authorization space down to the relevant volume of authorizations. With the exception of [rights amplification](#54-rights-amplification), every individual delegation MUST have equal or less scope from their delegator. Inside this content space, you can draw a boundary around some resource(s) (their type, identifiers, and paths or children), and their capabilities.
+ScopeB = [
+  { "with": "wnfs://alice.example.com/pictures/vacation/", "can": "wnfs/APPEND" };
+  { "with": "wnfs://alice.example.com/pictures/vacation/hawaii/", "can": "wnfs/OVERWRITE"}
+];
 
-As a practical matter, since scopes form a group, you can be fairly loose: order doesn’t matter, and merging resources can be quite broad since the more capable of any overlap will take precedence (i.e. you don’t need a clean separation).
+merge(ScopeA, ScopeA) == [
+   {"with": "wnfs://alice.example.com/pictures/", "can": "wnfs/APPEND"},
+   {"with": "wnfs://alice.example.com/pictures/vacation/", "can": "wnfs/OVERWRITE"}
```

Comment:
Should `OVERWRITE` extend to all vacation pictures after merging? Before the merge, `ScopeB` allowed overwriting Hawaii pictures, but not all vacation pictures.

Reply:
Good catch!